tests/mir-opt/pre-codegen/loops.int_range.PreCodegen.after.mir

```diff
@@ -51,64 +49,57 @@ fn int_range(_1: usize, _2: usize) -> () {
     }
 
     bb1: {
-        StorageLive(_11);
-        _5 = &mut _4;
-        StorageLive(_12);
-        StorageLive(_10);
-        StorageLive(_6);
-        _6 = &((*_5).0: usize);
-        StorageLive(_7);
-        _7 = &((*_5).1: usize);
         StorageLive(_8);
-        _8 = (*_6);
         StorageLive(_9);
-        _9 = (*_7);
-        _10 = Lt(move _8, move _9);
-        StorageDead(_9);
-        StorageDead(_8);
-        StorageDead(_7);
+        StorageLive(_7);
+        StorageLive(_5);
+        _5 = (_4.0: usize);
+        StorageLive(_6);
+        _6 = (_4.1: usize);
```

Comment:
I love to see all this collapsing, though!  Them no longer having their addresses taken should, I think, let `cg_ssa` generate them directly as SSA registers, saving a bunch of later work.